src/foam/comics/v2/DAOSummaryView.js

```diff
@@ -169,6 +173,8 @@ foam.CLASS({
             });
           }
 
+          var label = self.backLabel || `All ${config$browseTitle}`;
```

Comment:
It would be foamier to make `backLabel`an expression of `config$browseTitle`.

```JavaScript
    {
      class: 'String',
      name: 'backLabel',
      expression: function(config$browseTitle) {
        return 'All ' + config$browseTitle;
      }
    }
```

Reply:
thanks @TW80000 addressed in https://github.com/foam-framework/foam2/pull/3106